src/app/header/component.html

```diff
@@ -9,6 +9,9 @@
           <input #term id="searchbar" placeholder="Search Fall 2017" (keyup.enter)="search(term.value)"/>
         </span>
       </td>
+      <td class="sidelink" id="schedule-btn">
+        <a [routerLink]="['/courses']" [queryParams]="{ tags: ['Featured'] }">Topics<span class = "tiny">&nbsp; new!</span></a>
```

Comment:
`'featured'` should be lowercase

Reply:
Will correct when I can get to my computer